osc-dl.py

```diff
@@ -30,8 +33,10 @@
 
 # Get - downloads application
 get = subparser.add_parser(name='get', help="Download application.")
-# Send - downloads and sends application
-send = subparser.add_parser(name='send', help="Send application to Wii.")
+# IPSend - downloads and sends application over TCP/IP
+ipsend = subparser.add_parser(name='ipsend', help="Send application to Wii via TCP/IP.")
+# GeckoSend - downloads and sends application over USBGecko
+geckosend = subparser.add_parser(name='geckosend', help="Send application to Wii via USBGecko protocol.")
```

Comment:
It's probably better to keep the command "send" as is, for backwards compatibility purpose.
You could maybe make geckosend a mode of send for when something along the lines of "--gecko" is specified, or just rename ipsend back to just send.